script/Ciel/Day 6/06_01_CIEL06_1_2.txt

```diff
@@ -181,17 +188,23 @@ But as her older brother, I can't let her hear such a story. I would much rather
 [sha:8dc6f56686c12c23cb08f51187d905aa5b32dc9a]{
 -- Page 75, Offset 13354.
 -- 「そ、そう。最近知り合ったヤツなんだけど、一日だけでいいから観光名所を回りたいとか言い出して、気がつけば湾岸の埋め立て地あたりまで遠出したというか……」
-"Y-Yeah. I only met them recently, but they wanted to do some sightseeing around the famous places. Considering it was only for a day, they told me I should come along, and before I knew it, we had gone as far as the reclaimed land by the bay coast..."
+// 埋め立て地 can be either landfill (reclaimed land) or landfill (trash heap),
+// probably makes more sense that it's the former?
+"Y-Yeah. I only met them recently,
+ and they said they just wanted to do a quick day trip around the major tourist attractions,
+ but before I knew it we'd already made it as far as the reclaimed land over by the coast..."
 }
 [sha:3882ad42f801ba8bf07022b47f325fdb5249c2d7]{
 -- Page 76, Offset 13355.
 -- 「なるほど。久しぶりに会ったご友人に気軽に頼まれたぐらいで無断欠席および無断外泊をした、と。
-"I see. So you skipped school and stayed out without permission, all because you were nonchalantly asked to by a friend whom you haven't seen in a while.
+"I see. So a friend you haven't seen in a while casually asking you to go sightseeing with them is all it takes for you to blow off school and stay out without permission.
 }
 [sha:35c34940704b15f8d40c44032094f53c9e30ef85]{
 -- Page 76, Offset 13356.
 -- 先日の件といい、何の得にもならない人助けがお好きなんですね、兄さんは」
-Like what happened the other day, I see that you like to selflessly help other people, Nii-san."
+// I guess referring to the cover stor{y,ies} that Arihiko came up with at the end of d3?
+First what happened the other day, and now this...
```

Comment:
First,

Reply:
Kinda surprised about this one,  "First, what happened the other day" with the pause makes it sound more interrogative to me, as opposed to exasperated "first you saved a lost child in a bakery, and now you're showing people around town...". Am I just crazy?

Reply:
on re-review I agree with you. no comma